test/spec/suppressionList.spec.js

```diff
@@ -0,0 +1,136 @@
+var chai = require('chai')
+  , expect = chai.expect
+  , sinon = require('sinon')
+  , sinonChai = require('sinon-chai')
+  , nock = require('nock')
+  , SparkPost = require('../../lib/index');
+
+chai.use(sinonChai);
+
+describe('Suppression List Library', function() {
+  var client, suppressionList;
+
+  before(function () {
+    // setting up a client for all tests to use
+    var key = '12345678901234567890';
+
+    client = new SparkPost(key);
+    suppressionList = require('../../lib/suppressionList')(client);
+  });
+
+  it('should expose a public search method', function () {
```

Comment:
I'm not a fan of `.to.be.a.function` style tests. I feel like testing the functionality of these methods covers this